board/imgtec/pistachio_bub/fdt.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2016 Imagination Technologies
+ * Author: Francois Berder <francois.berder@imgtec.com>
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include <common.h>
+
+#if defined(CONFIG_OF_LIBFDT) && defined(CONFIG_OF_BOARD_SETUP)
+
+#include <winbond-otp.h>
+#include "otp.h"
+
+#define WIFI_STA_MAC_ADDRESS_OFFSET	0x1003
+#define WIFI_AP_MAC_ADDRESS_OFFSET	0x1009
+#define DCXO_OFFSET			0x2003
+
+DECLARE_GLOBAL_DATA_PTR;
+
+int __ft_board_setup(void *blob, bd_t *bd)
+{
+	ft_cpu_setup(blob, bd);
+
+	return 0;
+}
+int ft_board_setup(void *blob, bd_t *bd)
```

Comment:
Whats the need for the weak ref, no one is going to override us right? Just define ft_board_setup and call fixup_wifi directly from it as we have no cpu setup to do.